rocAL_pybind/amd/rocal/fn.py

```diff
@@ -176,8 +176,8 @@ def brightness_fixed(*inputs, brightness=1.0, brightness_shift=0.0, device=None,
         @return    Image with adjusted brightness
     """
     # pybind call arguments
-    kwargs_pybind = {"input_image": inputs[0], "is_output": False, "brightness": brightness, "brightness_shift": brightness_shift,
-                     "output_layout": output_layout, "output_dtype": output_dtype}
+    kwargs_pybind = {"input_image": inputs[0], "brightness": brightness, "brightness_shift": brightness_shift,
+                     "is_output": False, "output_layout": output_layout, "output_dtype": output_dtype}
```

Comment:
Is this change needed?

Reply:
Yes, it is corrected now according to the brightness fixed API
If not we were facing issues while testing